templates/bibrec.html

```diff
@@ -222,16 +314,18 @@ <h2>Read now or download (free!)</h2>
             <py:for each="n, e in enumerate (os.entries)">
           <py:if test="isinstance (e, bs.Cat) and e.rel not in ('start', )">
             <py:if test="e.header and old_header != e.header">
-              <h2 class="header">${e.header}</h2>
+              <h2 class="header" id="middle_section_header">${e.header}</h2>
             </py:if>
             <?python old_header = e.header ?>
 
             <div class="${e.class_}">
+			  <!-- Similar Books Links -->
               <a rel="nofollow"
                  href="${e.url}"
                  type="${e.type}"
                  charset="${e.charset}"
-                 accesskey="${str (n % 10)}">
+                 accesskey="${str (n % 10)}"
+                 id="similar-books-link">
```

Comment:
```suggestion
                 class="similar-books-link">
```
can't use ids because this is a repeated element